app/controllers/api/v0/shipments_controller.rb

```diff
@@ -28,19 +28,14 @@ def update
         authorize! :read, Spree::Shipment
         @shipment = @order.shipments.find_by!(number: params[:id])
         params[:shipment] ||= []
-        unlock = params[:shipment].delete(:unlock)
 
-        if unlock == 'yes'
-          @shipment.fee_adjustment.fire_events(:open)
-        end
+        @shipment.fee_adjustment.fire_events(:open)
 
         if @shipment.update(shipment_params)
           @order.updater.update_totals_and_states
         end
 
-        if unlock == 'yes'
-          @shipment.fee_adjustment.close
-        end
+        @shipment.fee_adjustment.close
```

Comment:
The only concern I have about this is that it closes adjustments even if there were open before this action. But the same would happen if you selected yes. So I guess that we can leave it as is.